### ISSUE TITLE
Move GeoIP library to composer dependency; update to GeoIPv2

### ISSUE THREAD

**Issue by asmecher** [pkp/pkp-lib#1833]
See: https://github.com/maxmind/GeoIP2-php

Ensure that any extra bumpf that comes with this library (e.g. `benchmark.php` and `admin/generate_geoipregionvars.php` from GeoIPv1) are not unnecessarily deployed.

**Comment by asmecher** [comment]
See https://blog.maxmind.com/2018/01/02/discontinuation-of-the-geolite-legacy-databases/ for a note on the discontinuation of GeoLite the legacy databases.

Moving to the 3.2.1 milestone for consideration -- this isn't confirmed yet.

**Comment by asmecher** [comment]
This also needs addressing in the `README.md` document, and the admin guide at https://docs.pkp.sfu.ca/admin-guide/en/statistics#configure-regional-data-tracking.

**Comment by asmecher** [comment]
(@ctgraham, have you encountered this?)

**Comment by ctgraham** [comment]
Yes, see also: https://forum.pkp.sfu.ca/t/ojs-3-usagestats-geoip-option-outdated/56537

In our case, we judged the internal geolocation-by-ip to be low value and disabled it in all journals, depending instead on external tooling like Google Analytics.

**Comment by NateWr** [comment]
Does a decision on this need to be made for 3.2.1 or can this be deferred?

**Comment by asmecher** [comment]
I think this can be deferred.

**Comment by asmecher** [comment]
(Related: https://forum.pkp.sfu.ca/t/is-it-still-possible-to-have-geographic-data-in-the-usage-reports)

**Comment by asmecher** [comment]
Closing in favour of #6895, which will fix/rewrite the underlying geolocation problem.
